ntirpc/rpc/pool_queue.h

```diff
@@ -47,7 +47,7 @@ struct poolq_entry {
 	TAILQ_ENTRY(poolq_entry) q;	/*** 1st ***/
 	u_int qsize;			/* allocated size of q entry,
 					 * 0: default size */
-	u_int qflags;
+	uint16_t qflags;
```

Comment:
This changes the size.  Currently, it looks okay, but if IOQ_FLAG_LOCKED, _UNLOCK, or _BALLOC is ever stored in here, it will overflow.  Should this be left as 32-bit?